src/Address/Validator/ERC20.php

```diff
@@ -25,7 +25,7 @@ public function validate(): bool
             return true;
         }
 
-        return $this->verifyChecksum();
+        return $this->verifyChecksum(substr($this->address->address(), 2));
```

Comment:
I would suggest to not use a non-static call on a static function.
To be honest I was surprised that php doesn't trigger E_STRICT warning on such a call any more.
But it may be confusing. E.g. when I see '->' in the code, I expect non-static method/property